api/signup.go

```diff
@@ -7,12 +7,14 @@ type SignupEnabledResponse struct {
 }
 
 type SignupRequest struct {
+	OrgName  string `json:"orgName"`
 	Name     string `json:"name"`
 	Password string `json:"password"`
 }
 
 func (r SignupRequest) ValidationRules() []validate.ValidationRule {
 	return []validate.ValidationRule{
+		validate.Required("orgName", r.OrgName),
```

Comment:
I think we probably want to `ValidateName` for the org name. That function assumes a field name of `"name"` so we'd have to do something like this:

```
rule := ValidateName(r.OrgName)
rule.Name = "orgName"
```

or refactor `ValidateName` to accept the field name as well.